internal/runbits/requirements/requirements.go

```diff
@@ -270,6 +271,17 @@ func (r *RequirementOperation) ExecuteRequirementOperation(
 	pg.Stop(locale.T("progress_success"))
 	pg = nil
 
+	// If applicable, output a list of additional dependencies (if any) being installed along with the
+	// requested package or bundle.
+	if operation == bpModel.OperationAdded &&
```

Comment:
Updates should also be considered. Updating a package can also bring in new dependencies.